### DISCUSSION TITLE
How to change hex constant representation from 'h' to '$'?

### DISCUSSION THREAD

**Discussion by TimRegeant** [NationalSecurityAgency/ghidra#6309] [General]
How would I get the listing window to show hex constants as $<hex>  instead of <hex>h?

ex)

dc.b  $FF
instead of
dc.b  FFh

**Comment by hippietrail** [comment]
On the old TI-99/4A the much rarer hex sigil `>` was used. It would be nice to have configurable and/or alternative syntaxes.
